node/node/start.go

```diff
@@ -149,12 +149,14 @@ func serve() error {
 		callback(err)
 		return err
 	}
+
+	// TODO: This code is not called
```

Comment:
why? node.Start is non-blocking, no?

Reply:
When I ran this code it never gets passed the Start so I never see the log message so as far as I could tell it was blocking